hbase-server/src/main/java/org/apache/hadoop/hbase/rsgroup/RSGroupUtil.java

```diff
@@ -31,9 +34,33 @@
 
   private static final Logger LOG = LoggerFactory.getLogger(RSGroupUtil.class);
 
+  public static final String RS_GROUP_ENABLED = "hbase.balancer.rsgroup.enabled";
```

Comment:
Not on all the time?

Reply:
You can disable the feature, then there will only be single default rs group.